repo_utils.py

```
@@ -0,0 +1,79 @@
+"""Utility for finding repository root and setting up import paths."""
+
+import sys
+from pathlib import Path
+
+
+def find_repo_root(start_path: Path = None) -> Path:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)